ams/server/src/main/java/com/netease/arctic/server/persistence/converter/Object2ByteArrayConvert.java

```diff
@@ -19,7 +20,7 @@ public void setParameter(PreparedStatement ps, int i, T parameter, JdbcType jdbc
       return;
     }
 
-    ps.setBytes(i, SerializationUtil.simpleSerialize(parameter).array());
+    ps.setBytes(i, CompressUtil.gzip(SerializationUtil.simpleSerialize(parameter).array()));
```

Comment:
Have you considered using Kryo? The size of Kryo is smaller then Java default serializer